### ISSUE TITLE
🐛 Abyssea

### ISSUE THREAD

**Issue by Freetate** [DerpyProjectGroup/topaz#2394]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)

## Branch affected by issue 

`base`

## Steps to reproduce

When the player is active in abyssea, if we restart the server, the player will be R0 and go online again, the player does not need to check the time, and will not be forced to be kicked out of abyssea, I think this should be a bug

## Expected behavior

The correct thing should be that when the player goes online, they will be given temporary access and a 5min countdown will be prompted.

**Comment by claywar** [comment]
Is the visitation status still active? If so, I believe this is expected behavior.  Note: it will still reduce in duration while offline

**Comment by Freetate** [comment]
No, when the player is in abyssea, if I force restart the server, when the player is online again, the time NPC has no effect on the current player, and he can always be in abyssea, no time limit

**Comment by Freetate** [comment]
Maybe the expression is not clear, what I want to say is that as long as I force restart the server, people who are still in abyssea at this stage, when the server is done, they are online again, they can be permanently in abyssea, as long as they do not leave voluntarily

**Comment by Freetate** [comment]
I have not tested whether it is possible to add atma normally in such a situation, because my local atma addition does not work, but they can be normal to fight monsters
